lib/std/sums.nim

```diff
@@ -8,6 +8,8 @@
 
 ## Accurate summation functions.
 
+{.deprecated: "use a nimble package instead.".}
```

Comment:
Perhaps link the package (or just the GitHub repo) or at least mention its name (`sums`).